shared/message/support/Gazebo/Gazebo.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package message.support.gazebo;
+
+/// \brief String message
+
+message StringMsg {
```

Comment:
Is this the string we send to/from gazebo? Can't we just send real messages?

Reply:
havent a clue